recipes/libcufile/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "libcufile" %}
+{% set version = "1.5.0.59" %}
+{% set platform = "linux-x86_64" %} # [linux64]
```

Think this is the line the linter is remarking on. This should fix it. Also this adds the other archs/OSes

```suggestion
 {% set platform = "linux-x86_64" %}  # [linux64]
 {% set platform = "linux-ppc64le" %}  # [ppc64le]
 {% set platform = "linux-sbsa" %}  # [aarch64]
 {% set platform = "windows-x86_64" %}  # [win]
 {% set extension = "tar.xz" %}  # [not win]
 {% set extension = "zip" %}  # [win]
```